### ISSUE TITLE
ILTIS tab not showing data

### ISSUE THREAD

**Issue by Yannan-Chen** [galizia-lab/pyview#28]
Hi, I'm following the below step and nothing shows up in ILTIS tab. Not sure if I'm anticipated to install it separately? 

Load data of one or more measurements ▶️ Transfer data to ILTIS ▶️ Browse data

My installation follows 'Installing VIEW as a normal user' in [wiki](https://github.com/galizia-lab/pyview/wiki/Installation-and-updating)

**Comment by ajkswamy** [comment]
Hi @Yannan-Chen. I need some more information to debug this:

1. Can you please try this again and check if the terminal/PowerShell in which you opened PyView shows a warning or error? If yes, please post it here.
2. Please tell me the data you loaded and the steps to replicate the situation?
3. If loading data succeeded, one should see a row of data in the "Data Manager" section of the GUI. Did your data load successfully?

**Comment by Yannan-Chen** [comment]
Hi @ajkswamy , of course.

1. Yes it gives one error as below
``` 
Traceback (most recent call last):
  File "C:\Users\Yannan\Anaconda3\envs\viewPy0\lib\site-packages\view\iltis_shell\main_shell.py", line 191, in import_data
    self.MainWindow.Options_Control.init_UI()
  File "C:\Users\Yannan\Anaconda3\envs\viewPy0\lib\site-packages\iltis\Widgets\Options_Control_Widget.py", line 36, in init_UI
    FormLayout.addRow('start/stop frames of stimuli',StimRegionWidget(self,'preprocessing','stimuli',self.Main.Options.preprocessing['nStimuli'],2,'f'))
  File "C:\Users\Yannan\Anaconda3\envs\viewPy0\lib\site-packages\iltis\Widgets\Options_Control_Widget.py", line 327, in __init__
    self.StimTable = ArrayWidget(parent, dict_name, param_name, rows, columns, dtype)
  File "C:\Users\Yannan\Anaconda3\envs\viewPy0\lib\site-packages\iltis\Widgets\Options_Control_Widget.py", line 258, in __init__
    self.horizontalHeader().setResizeMode(QtWidgets.QHeaderView.Stretch)
AttributeError: 'QHeaderView' object has no attribute 'setResizeMode'
```

2-3: For me I use OME/generic TIF to load one of my own 3D tif dataset and the 'Data Manager' shows the row. And 'Save Movie', 'Show foto1' work as expected.

**Comment by ajkswamy** [comment]
Hi @Yannan-Chen 

I was not able to replicate the issue with a generic TIF file that I have. 

Nonetheless, your error message and [this stackoverflow thread](https://stackoverflow.com/questions/26141161/pyqt4-code-not-working-on-pyqt5-qheaderview) lead me to suspect that this to be a minor bug in ILTIS. I have fixed this bug in ILTIS in the newly released version 1.6. 

PR #34 makes sure that PyView install ILTIS v1.6 when installing itself. Once #34 gets merged could you please reinstall PyView using the master (`pip install git+https://git@github.com/galizia-lab/pyview.git@master`) and check if you still get the error?

In case you get the error after reinstalling PyView, please share a sample TIF so that I can replicate the error and investigate further.

Thank you!

**Comment by Yannan-Chen** [comment]
Hi @ajkswamy,

Yes, it works now! Thanks.
